pkg/actor/validate_version.go

```diff
@@ -346,41 +421,67 @@ func IsContainerStatusImagePullBackoff(
 	pods, err := clientset.CoreV1().Pods(job.Namespace).List(ctx, metav1.ListOptions{
 		LabelSelector: labels.Set(job.Spec.Selector.MatchLabels).AsSelector().String(),
 	})
-	//TO DO: maybe we should check some k8s specific errors here
-	if err != nil {
-		return LogError("error getting pod in job", err, l)
+
+	if k8sErrors.IsNotFound(err) { // this is not an error
+		l.V(DEBUGLEVEL).Info("cannot find pods for vcheck job", "jobName", job.ObjectMeta.Name, "namespace", job.Namespace)
+		return err
```

Comment:
We need to return an error that we could not find the pod.  It is not an error until the backoff times out, and then we time out on this error.  We could return a custom error here, or do a `k8sErrors.IsNotFound(err)` further up so that we can log that we could not find the k8s object and we timed out.